demo/icefaces-applicant-portlet/pom.xml

```diff
@@ -57,10 +57,6 @@
 		</dependency>
 		<dependency>
 			<groupId>com.liferay.faces</groupId>
-			<artifactId>com.liferay.faces.alloy</artifactId>
```

Comment:
@ngriffin7a, I've removed the dependency on Alloy. But it's worth mentioning that I did not remove the dependency on Alloy Reslib from the `pluto` profile: https://github.com/liferay/liferay-faces-bridge-impl/pull/81/files#diff-3ab2cb346798a4c0c6e48f7abdc27bc1R31. I didn't remove it since we are not planning to release the Pluto version of the demos, and for testing purposes, reslib is necessary to style the demos correctly in Pluto.